### ISSUE TITLE
Special case WPF compiler errors

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Roslyn.Asserts#86]
```
Gu.Roslyn.Asserts.AssertException : Found errors.
CS1061 'MainWindow' does not contain a definition for 'InitializeComponent' and no accessible extension method 'InitializeComponent' accepting a first argument of type 'MainWindow' could be found (are you missing a using directive or an assembly reference?)
  at line 8 and character 17 in file C:\Git\Gu.Roslyn.Asserts\TestApps\WpfApp1\MainWindow.xaml.cs | this.↓InitializeComponent();
```

This is a false negative, `InitializeComponent()` is generated.
